src/internal/modules/charge-information/routes/non-chargeable.js

```diff
@@ -0,0 +1,131 @@
+const controller = require('../controllers/non-chargeable');
+const preHandlers = require('../pre-handlers');
+const { VALID_GUID } = require('shared/lib/validators');
+const Joi = require('@hapi/joi');
+const { charging } = require('internal/lib/constants').scope;
+const allowedScopes = [charging];
+
+module.exports = {
+  getNonChargeableReason: {
+    method: 'GET',
+    path: '/licences/{licenceId}/charge-information/non-chargeable-reason',
+    handler: controller.getNonChargeableReason,
+    options: {
+      auth: {
+        scope: allowedScopes
+      },
+      description: 'Select a non chargeable reason',
+      plugins: {
+        viewContext: {
+          activeNavLink: 'view'
+        }
+      },
+      validate: {
+        params: {
+          licenceId: VALID_GUID
+        },
+        query: {
+          form: VALID_GUID.optional(),
+          start: Joi.boolean().optional(),
+          returnToCheckData: Joi.number().default(0).allow(0, 1)
```

Comment:
I feel like `start` and `returnToCheckData` should be consisten(ie a boolean or an integer) as they are serving the same prupose. I used an integer for `returnToCheckData` to match the pattern in the billing module `back` pattern. I'm happy to chat this over, if you'd like :)